mm2src/mm2_tests.rs

```diff
@@ -7488,7 +7488,8 @@ mod wasm_bindgen_tests {
 
     wasm_bindgen_test_configure!(run_in_browser);
 
-    #[wasm_bindgen_test]
+    // TODO uncomment it, when the P2P supports in-memory transport
+    // #[wasm_bindgen_test]
```

Comment:
I commented out it when the in-memory transport hadn't been supported yet :D
I will check if this test is green. Thank you!

Reply:
I've replace `wasm_bindgen_tests::test_swap` with `trade_test_rick_and_morty`. This test doesn't check the swap stats and statuses because the swap db is not fully functional yet.